rules/falco_rules.yaml

```diff
@@ -3113,8 +3110,7 @@
   condition: >
     spawned_process and container
     and container.privileged=true
-    and proc.name=mount
-    and not mount_info
+    and evt.type = mount
```

Comment:
this is not going to work as the `spawend_process` indicates `evt.type=execve`, and simply detect mounts inside a container may trigger a lot FP which I haven't explored yet.

Reply:
I see. I will test this rule to my testbed to know how many FPs are detected.
So, if you have a solution to reduce FP, cloud you tell me?